ompi/runtime/ompi_mpi_params.c

```diff
@@ -263,12 +263,11 @@ int ompi_mpi_register_params(void)
         ompi_rte_abort(1, NULL);
     }
 
-    ompi_add_procs_cutoff = 1024;
```

Comment:
@hjelmn I'm not sure why you say that this extra assignment is necessary (in https://github.com/jsquyres/ompi/commit/cd5bad832aad5ae33108d8d7d9a8b939bcae9af4#commitcomment-15887486).  Even if we come through registration a 2nd time, we don't want to reset the default if the user already set a different value.  What am I missing here?

Regardless, we should either be doing this extra assignment for *all* MCA params, or *no* MCA params -- if there's a reason to do the extra assignment for this MCA param, that same reason should apply to *all* MCA params, right?